appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -2743,4 +2755,18 @@ public void run(Objectify datastore) {
       throw CrashReport.createAndLogError(LOG, null, collectUserErrorInfo(userId), e);
     }
   }
+
+  /*
+   * Determine which GCS Bucket to use based on filename. In particular
+   * APK files go in a bucket with a short TTL, because they are really
+   * temporary files.
+   */
+  private final String getGCSBucketToUse(String fileName) {
```

Comment:
This could be made `static`. Also, under the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) the name should be `getGcsBucketToUse`.